initial_setup/tui/spokes/eula.py

```diff
@@ -0,0 +1,128 @@
+"""EULA TUI spoke for the Initial Setup"""
+
+import gettext
+import codecs
+import logging
+
+from pyanaconda.ui.tui.spokes import NormalTUISpoke
+from simpleline.render.widgets import TextWidget, CheckboxWidget, SeparatorWidget
+from simpleline.render.containers import ListRowContainer
+from simpleline.render.screen import UIScreen, InputState
+from simpleline.render.screen_handler import ScreenHandler
+from pyanaconda.ui.common import FirstbootOnlySpokeMixIn
+from initial_setup.product import get_license_file_name, eula_available
+from initial_setup.common import LicensingCategory
+from pyanaconda.core.constants import FIRSTBOOT_ENVIRON
+from pykickstart.constants import FIRSTBOOT_RECONFIG
+
+log = logging.getLogger("initial-setup")
+
+_ = lambda x: gettext.ldgettext("initial-setup", x)
+N_ = lambda x: x
+
+__all__ = ["EULAspoke"]
+
+
+class EULAspoke(FirstbootOnlySpokeMixIn, NormalTUISpoke):
```

Comment:
The name should be probably `EULASpoke` to keep it consistent with other spokes.